syscalls/src/lib.rs

```diff
@@ -1609,19 +1613,34 @@ declare_builtin_function!(
             invoke_context.get_check_aligned(),
         )?;
 
-        let calculation = match group_op {
-            ALT_BN128_ADD => alt_bn128_addition,
-            ALT_BN128_MUL => alt_bn128_multiplication,
-            ALT_BN128_PAIRING => alt_bn128_pairing,
+        let result_point = match group_op {
+            ALT_BN128_G1_ADD_BE => {
+                alt_bn128_versioned_g1_addition(VersionedG1Addition::V0, input, Endianness::BE)
+            }
+            ALT_BN128_G1_MUL_BE => {
+                alt_bn128_versioned_g1_multiplication(
+                    VersionedG1Multiplication::V0,
```

Comment:
```suggestion
                    VersionedG1Multiplication::V1,
```
V0 is for 128 bytes input

Reply:
Ah good catch! Forgot that this feature is already active.  Should be fixed now.

Reply:
@LStan SIMD-334 is actually not active, correct? I think we should revert this back to V0 and then on a follow-up PR, upgrade to V1 behind a feature gate.

Reply:
It is a good point regarding `target(solana)`. I was pinged by the fd team that they had some issues with the semantic versioning in Cargo for `solana-bn254` due to breaking changes. It is unclear if they use `solana-bn254` as an actual dependency, but if so, then the best path forward seems to be for them to use the versioned function as well. Waiting to hear back from them since they are in the US timezone.